tensorflow/lite/profiling/profile_summarizer_test.cc

```diff
@@ -86,15 +84,6 @@ class SimpleOpModel : public SingleOpModel {
   int output_;
 };
 
-void SimpleOpModel::Init(
```

Comment:
I don't fully understand this removal? Why?

Reply:
unused code cleanup.

Reply:
`m.Init(RegisterSimpleOp);` is called below, in ProfileSummarizerTest.Interpreter.

Reply:
m.Init(RegisterSimpleOp); this is L76 method invocation, after remove this code also test case run is success.

Reply:
That's because that test is behind a build define.

Reply:
so how to deal this now?

Reply:
I've removed the build define usage, so we can close this PR.